clang/test/dpct/cub/warplevel/warpreduce.cu

```diff
@@ -68,6 +68,47 @@ __global__ void ReduceKernel(int* data) {
   data[threadid] = output;
 }
 
+
+//CHECK: void ReduceKernel_Max(int* data, const sycl::nd_item<3> &item_ct1) {
+//CHECK:  int threadid = item_ct1.get_local_id(2);
+//CHECK:  int input = data[threadid];
+//CHECK:  int output = 0;
+//CHECK:  output = sycl::reduce_over_group(item_ct1.get_sub_group(), input, sycl::maximum<>());
```

Comment:
Use CHECK-NEXT here maybe better to avoid ambiguous when check.

Reply:
Agree . added CHECK-NEXT for the other tests as well. Thanks